java/code/src/com/redhat/rhn/common/db/datasource/xml/PackageEvr_queries.xml

```diff
@@ -0,0 +1,13 @@
+<datasource_modes>
```

Comment:
Nitpick: if these queries are only used for testing, they could be added to the `test_queries.xml` file

Reply:
Fixed